### ISSUE TITLE
AttributeError: type object 'JobInfrastructureConfigurationDetails' has no attribute 'JOB_INFRASTRUCTURE_TYPE_ME_STANDALONE'

### ISSUE THREAD

**Issue by jrgauthier01** [oracle/accelerated-data-science#5]
When executing 
```
from ads.jobs import Job
from ads.jobs import DataScienceJob, ScriptRuntime

job_name = 'attrition-model-training-job'
```
I'm getting : 

```
AttributeError: type object 'JobInfrastructureConfigurationDetails' has no attribute 'JOB_INFRASTRUCTURE_TYPE_ME_STANDALONE'
```
in ADS 2.5.9.

**Comment by qiuosier** [comment]
This is caused by the fact that ADS is using a variable that is not available in the older oci python SDK. Updating oci to 2.59.0 or higher (`pip install oci>=2.59.0`) will solve the issue. We will add this into the requirements of ADS in the next release.

**Comment by qiuosier** [comment]
This is fixed in 2.5.10: https://github.com/oracle/accelerated-data-science/releases/tag/v2.5.10
